opencolorio_config_aces/config/reference/generate/config.py

```diff
@@ -143,8 +144,9 @@
 
 LOOK_NAME_SUBSTITUTION_PATTERNS = {
     # TODO: Implement support for callable patterns.
-    # The following one should be a dedicated definition/callable.
-    'BlueLightArtifactFix': 'Blue Light Artifact Fix'
+    # The following ones should be a dedicated definition/callable.
+    'BlueLightArtifactFix': 'Blue Light Artifact Fix',
+    'GamutCompress': 'Gamut Compress',
```

Comment:
I think this should be called "ACES 1.3 Reference Gamut Compression" rather than simply "Gamut Compress".